onnx/defs/experiments/defs.cc

```diff
@@ -363,3 +363,87 @@ OPERATOR_SCHEMA(ATen)
 Experimental allowing ATen operations to be accessed directly from Caffe2
 to allow for quick prototyping when ONNX is missing standard versions of
 and op)DOC");
+
+
+OPERATOR_SCHEMA(ImageScaler)
+    .SetSupportLevel(SupportType::EXPERIMENTAL)
+    .NumInputs(1)
+    .NumOutputs(1)
+    .AllowConsumed({{0, 0}})
+    .SetDoc(R"DOC(Scale and bias the input image. Bias values are stored in 
+the same ordering as the image pixel format.)DOC")
+    .Attr("bias", "Bias applied to each channel, same size as C.", AttrType::FLOATS)
+    .Attr("scale", "(float, default 1.0) the scale to apply.", AttrType::FLOAT)
+    .Input(0, "input", "Input tensor of shape [N,C,H,W]", "T")
+    .Output(0, "output", "Result, has same shape and type as input", "T")
+    .TypeConstraint(
+        "T",
+        {"tensor(float16)", "tensor(float)", "tensor(double)"},
+        "Constrain input and output types to float tensors.");
+
+OPERATOR_SCHEMA(MeanSubtraction)
```

Comment:
Is this covered by Sub (by putting the attribute "image" as the second input)?

Reply:
Agree removed.